source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -518,6 +518,7 @@ ShaderGraphPtr ShaderGraph::createSurfaceShader(
             if (nodeInputValue)
             {
                 inputSocket->setValue(nodeInputValue);
+                input->setBindInput();
```

Comment:
This flag is set for 1.37 but not for 1.38 so colorspace directly specified on a shader would be ignored.